core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -116,10 +120,27 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
   contentTypeIsSetOfStrings: boolean = false;
   contentTypeIsUnicode: boolean = false;
   lastSuggestionToReview: boolean = false;
+  firstSuggestionToReview: boolean = true;
   resolvingSuggestion: boolean = false;
   reviewable: boolean = false;
   canEditTranslation: boolean = false;
   userIsCurriculumAdmin: boolean = false;
+  isContentExpanded: boolean = false;
+  isContentOverflowing: boolean = false;
+  isTranslationExpanded: boolean = false;
+  isTranslationOverflowing: boolean = false;
+  @ViewChild('contentPanel')
```

Comment:
Nit: Add new line above to be consistent with below.

Reply:
Done